zapcore/facility_test.go

```diff
@@ -113,7 +113,7 @@ func TestWriterFacilitySyncsOutput(t *testing.T) {
 	}
 }
 
-func TestWriterFacilityWriteEntryFailure(t *testing.T) {
```

Comment:
Can we add another test case here with the short writer from the testutils package?